samples/SocialSample/Startup.cs

```diff
@@ -58,6 +61,17 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             {
                 options.ClientId = "560027070069-37ldt4kfuohhu3m495hk2j4pjp92d382.apps.googleusercontent.com";
                 options.ClientSecret = "n2Q-GEw9RQjzcRbU3qhfTj8f";
+                options.Events = new OAuthEvents()
+                {
+                    OnRemoteError = ctx =>
```

Comment:
That's always the fallback option, this is supposed to only be for any specific targetted exceptions that can be special case handled.  We haven't fully worked through those yet, but yes the idea is a few common error paths should be handled somewhat more gracefully than today (which is an admittedly low bar :)